content/docs/add-react-to-a-website.md

```diff
@@ -152,30 +152,21 @@ You can play with JSX using [this online converter](http://babeljs.io/repl#?babi
 
 ### Add JSX to a Project
 
-If you want to add JSX to your project, follow these three steps:
+If you want to add JSX to your project, follow these steps:
 
-1. [Install Node.js](https://nodejs.org/)
-2. **Don't miss this step:** Run `npm init -y` in your project folder
-    - Tip: If you get an **error message** starting with "Invalid name", rename the project folder to only contain lowercase ASCII letters or hyphens (e.g. `my-project`), and try again.
-3. Run `npm install babel-cli@6 babel-preset-react-app@3`
+1. [Install Node.js](https://nodejs.org/) version 5 or above
+1. Run `npx create-react-babel` in a terminal
+   - Follow the prompts to create a package.json with Babel
+   - Tip: If you get an **error message** starting with "Invalid name", rename the project folder to only contain lowercase ASCII letters or hyphens (e.g. `my-project`), and try again.
+1. Run `npm install`
+1. Move your component files into `src`
```

Comment:
it actually prompts for the input and output directories but `src` is the default